tensorboard/program.py

```diff
@@ -348,35 +350,65 @@ def __init__(self, wsgi_app, flags):
     self._flags = flags
     host = flags.host
     self._auto_wildcard = False
```

Comment:
It's outside the changes in this PR, but can you add a comment to this field?

Reply:
One better: I’ve moved this down to the logic that actually initializes
`_auto_wildcard`, where it’s already documented (clearly, IMO).